app/models/event.rb

```diff
@@ -40,6 +40,23 @@ def logo_delete
     @logo_delete || '0'
   end
 
+  def featured_projects_favorites_stats
```

Comment:
:raised_hands: sweeet! Can you please add another method similar to this which returns all projects favorited during an event, rather than just the featured projects favorited during an event? :+1: